examples/python/bitmap_operation.py

```diff
@@ -0,0 +1,60 @@
+''' 
+Example of how to use python wrappers of penguinV to save a gray scale version of a bitmap image.
+'''
+
+# Module files for penguinV are in separate folder, so we use the sys module to add that location
+# to the places that python will look for the module.
+
+import sys
+sys.path.insert(0, '..\\..\\src\\python')
+
+# Now get the module.
+
+import penguinV 
+
+def saveGrayScaleCopy(filenameToOpen, filenameToSave):
+    ''' Saves a Gray Scale Thresholded version of a bitmap file.
+    
+    Parameters
+    ----------
+    filenameToOpen : String
+        Name of the bitmap to open.
+
+    filenameToSave : String
+        Name of the resulting bitmap.
+    '''
+
+    image = penguinV.Load(filenameToOpen)
+    
+    if image.empty():
+        raise penguinV.ImageException('penguinV.Load()', 'Problem loading file ' + filenameToOpen)
```

Comment:
Why don't we just use `print` function and call normal return. In such situation we don't need an extra exception. What do you think?

Sorry for late reply, github was down for me almost whole day.

Reply:
I was just trying to make the python version as close to the .cpp version as possible. Shouldn't we at least keep the `penguinV.ImageException` to showcase using the exceptions? If you think its unnecessary, then I'm totally fine with removing it.

I was also having a lot of trouble with github yesterday.

Reply:
Well, it's just an example. It should just work as `cpp` version but nobody demands to be an exact copy of it. I think we just go with what we have.

Github is still having problems.